### ISSUE TITLE
Hide bookmarks vanity count

### ISSUE THREAD

**Issue by Shpigford** [thomaswang/minimal-twitter#180]
Would love a toggle to hide the bookmarks count.

![CleanShot 2023-05-17 at 09 06 51@2x](https://github.com/typefully/minimal-twitter/assets/35243/0ffdca19-c505-4123-b6b8-18a9c1ea3c1e)

**Comment by Dineshs91** [comment]
Where is this visible? I am not able to find it in the current UI

**Comment by Dineshs91** [comment]
That's right. I will file a task to hide bookmark count as well when `Engagements Under Posts` option is selected.
